services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -887,21 +914,37 @@ export class NoticeOfWorkApplication extends Component {
                 <LoadingWrapper condition={this.state.isTabLoaded}>
                   <div className={this.renderFixedHeaderClass()}>
                     <div className="inline-flex">
-                      <h2 className="padding-md">
-                        Consultation
-                        <CoreTooltip
-                          title="This page allows you to identify and download the files that need to be included in the package for first nations consultations.
-                          You may track progress on the Consultation reports and tracking system (CRTS).
-                          When responses are received you can upload them by clicking on “Add Reviewer” .
-                          Finish this stage by clicking on “Complete Process” when all responses have been received. If you need to make changes later, click “Resume Consultation process”"
-                        />
+                      <h2 className="tab-title">
+                        <Popover
+                          placement="topLeft"
+                          content="This page allows you to identify and download the files that need to be included in the package for first nations consultations.
+            You may track progress on the Consultation reports and tracking system (CRTS).
+            When responses are received you can upload them by clicking on “Add Consultation” .
```

Comment:
Can we fix all instances of these quotes in the project? :D
`“Add Consultation”` --> `"Add Consultation"`

Reply:
fixed